autotune/hpo-keras/autotune_gpu_t9ksched.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: tensorstack.dev/v1beta1
+kind: AutoTuneExperiment
+metadata:
+  name: keras-mnist-autotune
+spec:
+  maxExecSeconds: 3600
+  maxTrialNum: 20
+  trialConcurrency: 3
+  aimd:
+    secret: api-key-aimd
+    folder: 
+  storage: 100Mi
+  searchSpace: |-
+    {
+      "batch_size": {"_type": "choice", "_value": [16, 32, 64, 128]},
+      "learning_rate": {"_type": "choice", "_value": [0.0001, 0.001, 0.01, 0.1]},
+      "conv_channels1": {"_type": "choice", "_value": [16, 32, 64, 128]}
+    }
+  trainingConfig:
+    type: tensorflow
+    scheduler:
+      t9kScheduler:
+        queue:
```

Comment:
1. use a default queue name `default` 
2. use a non-zero value for `priority`, say `10`

Do the same for all other `scheduler` fields

Reply:
modified